socketIO_client/transports.py

```diff
@@ -84,9 +87,19 @@ def recv_packet(self):
             self._http_url,
             params=params,
             **self._kw_get)
-        for engineIO_packet in decode_engineIO_content(response.content):
-            engineIO_packet_type, engineIO_packet_data = engineIO_packet
-            yield engineIO_packet_type, engineIO_packet_data
+        try:
+            for engineIO_packet in decode_engineIO_content(response.content, self._bw_comp):
+                engineIO_packet_type, engineIO_packet_data = engineIO_packet
+                yield engineIO_packet_type, engineIO_packet_data
+        except UnicodeDecodeError as error:
+            if self._bw_comp:
+                raise error
```

Comment:
You should be able to get rid of the error variable and just use raise without any args.